### PR TITLE
Post page

### PR DESCRIPTION
**Pull request by Priya-Nigam**
Created baseline for post page for mobile, where user can access gallery or camera to take a picture and "upload" it to their account with a caption and triggers.

### CODE REVIEW

File: Post/Post.js [hunk 1]
```diff
@@ -1,40 +1,139 @@
...
+
+    //if no camera possible (web) or no gallery permission, say sorry! no access to gallery or camera
+    if (hasCameraPermission === null || hasGalleryPermission === false) {
+        return (<View style = {KIC_Style}>
+            <Image
+                style={{ width: 180, height: 180, alignItems: "center", resizeMode: 'contain' }}
+                source={require('../assets/kic.png')}
+            />
+            <Text>
+                Sorry! No access to gallery or camera. This feature is only available on mobile!
```
Comment (tug59430) [inline Post/Post.js:71]: Thanks for handling this case too!

File: User-Authentication/SignUp.js [hunk 2]
```diff
@@ -6,119 +6,129 @@ import React from 'react';
...
-    }
-}; 
+    const [firstName, setFirstName] = useState("");
+    const [lastName, setLastName] = useState("");
+    const [username, setUserName] = useState("");
+    const [email, setEmail] = useState("");
+    const [password1, setPassword1] = useState("");
+    const [password2, setPassword2] = useState("");
+
+    const handleSubmit = evt => {
```
Comment (tug59430) [inline User-Authentication/SignUp.js:28]: Looks comprehensive!

**Review by tug59430 — APPROVED** [review]
New changes look good!

**Review by jaimer7 — APPROVED** [review]
The styling looks really good and its impressive how much you got done for this baseline!

**Pull request merged**